src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4652,16 +4652,13 @@ static InitialSessionState()
                 #endregion
             };
 
-            if (ExperimentalFeature.IsEnabled(ExperimentalFeature.PSNativeCommandErrorActionPreferenceFeatureName))
-            {
-                builtinVariables.Add(
-                    new SessionStateVariableEntry(
-                        SpecialVariables.PSNativeCommandUseErrorActionPreference,
-                        value: true,    // when this feature is changed to stable, this should default to `false`
-                        RunspaceInit.PSNativeCommandUseErrorActionPreferenceDescription,
-                        ScopedItemOptions.None,
-                        new ArgumentTypeConverterAttribute(typeof(bool))));
-            }
+            builtinVariables.Add(
+                new SessionStateVariableEntry(
+                    SpecialVariables.PSNativeCommandUseErrorActionPreference,
+                    value: true,    // when this feature is changed to stable, this should default to `false`
```

Comment:
You missed this important comment :)

```suggestion
                    value: false,
```